ext/config/bolt-server.conf

```diff
@@ -0,0 +1,6 @@
+bolt-server: {
+    port: 8144
+    ssl-cert: cert.pem
```

Comment:
Are these useful defaults?

Reply:
yeah I was wondering they should be commented out rather then pointing to certs we dont' expect to exist. If orch does this it's fine for consistency.

Reply:
:+1: https://github.com/puppetlabs/orchestrator/blob/johnson/resources/ext/config/conf.d/orchestrator.conf is what I'm copying just for full disclosure!